code/client/src/core/ui/vehicle_debug.cpp

```diff
@@ -20,6 +20,8 @@ namespace MafiaMP::Core::UI {
 
         if (currentCar) {
             auto currentVehicle = currentCar->GetVehicle();
+            ImGui::Text("Car Ptr: %p", currentCar);
+            ImGui::Text("Vehicle Ptr: %p", currentVehicle);
```

Comment:
These are already at the bottom : https://github.com/MafiaHub/MafiaMP/pull/90/files#diff-9b678781c9bd580b6852983396c0fa61aee6c99734fc0e99a0f9c8a5f1419d97R178-R179